write locks documentation (#25)
qdrant/docs: qdrant/v0.11.0.x/administration.md

---
title: Administration
weight: 21
---

> Qdrant has administration tools. It is useful when you get access to your qdrant instance or run qdrant in the cloud.

Reviewer: ```suggestion
Qdrant exposes administration tools which enable to modify at runtime the behavior of a running instance without changing its configuration manually.
```

Author: Thanks, your option is better =)

## Locking

> Qdrant has locking functionality when you want to turn off some features from qdrant process. Locking is not persistent and you have to lock again after restarting qdrant.

Author: Right

> Qdrant has locking functionality when you want to turn off some features from qdrant process. Locking is not persistent and you have to lock again after restarting qdrant.

Reviewer: ```suggestion
A locking API enables users to turn off restrict the possible operations on a qdrant process.
It is important to mention that:
- The configuration is not persistent therefore it is necessary to lock again following a restart.
- Locking applies to a single node only. It is necessary to call lock on all the desired nodes in a distributed deployment setup.
```

Lock request sample:

```http
POST /locks
{
    "error_message": "write is forbidden",
    "write": true
}
```

Write flags enables/disables write lock.

> If write lock is enabled, qdrant doesn't allow creating collections or adding new data to storage.

Reviewer: ```suggestion
If the write lock is set to true, qdrant doesn't allow creating new collections or adding new data to the existing storage.
```

> But deletion or updating is not forbidden.

Reviewer: ```suggestion
However deletion operations or updates are not forbidden under the write lock.
```

> It is useful when qdrant uses too much disk space and administrator wants to limit disk usage.

Reviewer: ```suggestion
This feature enables administrators to prevent a qdrant process from using more disk space while permitting users to search and delete unnecessary data.
```

---
title: Administration
weight: 21
---

> Qdrant exposes administration tools which enable to modify at runtime the behavior of a running instance without changing its configuration manually.

Reviewer: ```suggestion
Qdrant exposes administration tools which enable to modify at runtime the behavior of a qdrant instance without changing its configuration manually.
```

## Locking

> A locking API enables users to turn off restrict the possible operations on a qdrant process.

Reviewer: ```suggestion
A locking API enables users to restrict the possible operations on a qdrant process.
```
